.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 11
+node_js: node
```

Comment:
I’ve been setting this to 10 on all our projects because otherwise a new version of Node can break the build… I’d rather our builds be a little more stable.